neo/Network/P2P/Message.cs

```diff
@@ -152,6 +154,8 @@ internal static int TryDeserialize(ByteString data, out Message msg)
                 Command = (MessageCommand)header[1],
                 _payload_compressed = length <= 0 ? new byte[0] : data.Slice(payloadIndex, (int)length).ToArray()
             };
+            if (!Enum.IsDefined(typeof(MessageCommand), msg.Command))
```

Comment:
We don't need to check it here. If we add new command in the future, it can be compatible with the current version.